src/js/components/Tabs/Tabs.js

```diff
@@ -75,6 +77,19 @@ const Tabs = forwardRef(
       this,
     );
 
+    const tabsHeaderStyles = {};
+    if (theme.tabs.header && theme.tabs.header.border) {
+      let borderColor =
+        theme.tabs.header.border.color || theme.global.control.border.color;
+      borderColor = normalizeColor(borderColor, theme);
+
+      tabsHeaderStyles.border = {
+        side: theme.tabs.header.border.side,
+        size: theme.tabs.header.border.size,
```

Comment:
You may have a case where `theme.tabs.header.border` exist, but `theme.tabs.header.border.size` doesn't. Same for `side`.

Reply:
Yes, in these cases the values would just become undefined. which is fine right?

Reply:
Not sure how a border that has color but no size/side will react, it doesn't sound right to me. Can you compare this use case with Box border and make sure it is consistent?

Reply:
Yes, let me check it out

Reply:
If any of these values is undefined, then it will fallback to the defaults defined in `borderStyle` in `border.js` (consistent with Box):
```
const color = normalizeColor(data.color || 'border', theme);
const borderSize = data.size || 'xsmall';
const style = data.style || 'solid';
const side = typeof data === 'string' ? data : data.side || 'all';
```
https://github.com/grommet/grommet/blob/master/src/js/utils/border.js#L7

This is because `StyledTabsHeader` is a `Box`, so the behavior matches that of Box.